File: tugas/java/Disk.java
public class Disk {

    // atribut
    private String type;
    private int capacity;
    private int price;

    // constructor
    public Disk(){}

    public Disk(String type, int capacity, int price) {
        this.setType(type);
        this.setCapacity(capacity);
        this.setPrice(price);
    }

    // setter n getter
    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return price;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getType() {
        return type;
    }
}

File: tugas/cpp/Disk.cpp
#include <bits/stdc++.h>

using namespace std;

class Disk {
    private: 
        // atribut
        string type;
        int capacity, price;

    public:
        // constructor
        Disk(){
            this->type = "-";
            this->capacity = 0;
            this->price = 0;
        }

        Disk(string type, int capacity, int price){
            this->type = type;
            this->capacity = capacity;
            this->price = price;
        }

        // setter n getter
        void setType(string type){
            this->type = type;
        }

        string getType(){
            return this->type;
        }

        void setCapacity(int capacity){
            this->capacity = capacity;
        }

        int getCapacity(){
            return this->capacity;
        }

        void setPrice(int price){
            this->price = price;
        }

        int getPrice(){
            return this->price;
        }

        ~Disk(){};
};